CSA.md

```diff
@@ -97,19 +99,19 @@ Capitalized and highlighted words have the meanings given on the Cover Page or i
 
```

Comment:
Same question about bolding for the text of 9.1 and 9.2, can those sentences be bolded (but no need to make the text in head_3 style bolded)?

Reply:
yeah, I think you figured it out with the **